tests/business/jupyterpythonloop_test.py

```diff
@@ -28,6 +28,7 @@ async def test_run(
             "count": 1,
             "user_spec": {"username_prefix": "testuser", "uid_start": 1000},
             "scopes": ["exec:notebook"],
+            "options": {"idle_time": 2, "settle_time": 2, "max_executions": 2},
```

Comment:
If you make the change mentioned above, you can, here, set the settle time and pause between cells to 0 but leave the idle time at the end of each cycle at the default, and that will work well with the test and speed it up even further.

Reply:
Split lab_idle and execution_idle apart.